rclpy/test/test_logging.py

```diff
@@ -104,8 +104,8 @@ def test_log_throttle(self):
                 throttle_duration_sec=1,
                 throttle_time_source_type='RCUTILS_STEADY_TIME',
             ))
-            time.sleep(0.3)
-        self.assertEqual(message_was_logged, [True] + [False] * 3 + [True])
+            time.sleep(0.4)
```

Comment:
With the new sleep value one of the wake up cases is "by number" exactly happening at the same time as the throttle duration (5 * 0.4 = 2.0). I though the idea of the previous value was that this is not the case?

Reply:
the last call will be at t=1.6s

Reply:
I was thinking about if ever one more call is made it might get hairy with the timing. For now that should be good enough, I agree.